cmd/gapit/server_performance.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"github.com/google/gapid/core/app"
+	"github.com/google/gapid/core/log"
+	"github.com/google/gapid/gapis/service"
+	gapidPath "github.com/google/gapid/gapis/service/path"
+)
+
+type serverPerformanceVerb struct{ ServerPerformanceFlags }
+
+func init() {
+	verb := &serverPerformanceVerb{
+		ServerPerformanceFlags{LoopCount: 1},
+	}
+	app.AddVerb(&app.Verb{
+		Name:      "server_performance",
+		ShortHelp: "Test the performance of GAPIS.",
```

Comment:
Maybe be more precise in the short help, e.g.: "Test the performance of replay generation in GAPIS."

Reply:
I was going to comment the same. This only tests replay generation so let's be clear to avoid confusion.